gateway/api/services/authentication/quantum_platform.py

```diff
@@ -0,0 +1,175 @@
+"""This service will manage the access to the 3rd party end-points in Quantum platform."""
+
+
+import logging
+from typing import List
+from django.conf import settings
+import requests
+
+from api.utils import remove_duplicates_from_list, safe_request
+
+logger = logging.getLogger("gateway.services.quantum_platform")
+
+
+class QuantumPlatformService:
+    """
+    This class will manage the different access to the different
+    end-points that we will make us of them in this service.
+    """
+
+    def __init__(self, authorization_token):
+        self.auth_url = (
+            f"{settings.QUANTUM_PLATFORM_API_BASE_URL}/api/users/loginWithToken"
+        )
+        self.verification_url = f"{settings.QUANTUM_PLATFORM_API_BASE_URL}/api/users/me"
+        self.instances_url = f"{settings.IQP_QCON_API_BASE_URL}/network"
+        self.authorization_token = authorization_token
+        self.access_token = None
+
+    def _get_network(self, access_token: str):
```

Comment:
This is something very interesting @paaragon . If I use `__get_network` here I cannot mock the method because as a private method the test cannot access to the method.

I didn't remember this so probably we should stop using that nomenclature, what do you think?